dom.bs

```diff
@@ -962,6 +966,7 @@ dictionary EventListenerOptions {
 dictionary AddEventListenerOptions : EventListenerOptions {
   boolean passive = false;
   boolean once = false;
+  AbortSignal? signal;
```

Comment:
I think the IDL should not be nullable. Since this is not marked `required`, you can already omit it (or pass undefined). Additionally allowing passing `null` is not great, and is something we've avoided with other APIs.

Reply:
So just `AbortSignal signal` would also work ? I followed the one usage example I know ([in fetch](https://fetch.spec.whatwg.org/#requestinit)) - should I attempt to also fix this there?

Reply:
It would work, but hmm, that inconsistency is interesting. Maybe hold off on changes until @annevk chimes in...